pkg/autoscaler/multiscaler.go

```diff
@@ -166,7 +166,7 @@ func (m *MultiScaler) Get(ctx context.Context, namespace, name string) (*Decider
 	}
 	scaler.mux.RLock()
 	defer scaler.mux.RUnlock()
-	return (&scaler.decider).DeepCopy(), nil
+	return scaler.decider.DeepCopy(), nil
```

Comment:
Why still need `DeepCopy` here?

Reply:
I guess here we don't, since we follow `don't modify informer copy`.